gulp/lint.js

```diff
@@ -13,7 +14,12 @@ gulp.task('lint', ['lint:sass', 'lint:javascript']);
 
 gulp.task('lint:sass', function() {
   return gulp.src(PATHS)
-    .pipe(scssLint());
+    .pipe(plumber())
```

Comment:
`plumber()` make the script ignoring `sass-lint` error and continue the compilation